playbooks/cloud-provider-openstack-acceptance-test-e2e-conformance/run.yaml

```diff
@@ -24,6 +24,17 @@
             extraArgs:
               "cloud-provider": "external"
 
+    - name: Install python 3.7 on this Xenial host which is needed by post job
+      shell:
+        executable: /bin/bash
+        cmd: |
+          set -ex
+
+          apt install -y software-properties-common
+          add-apt-repository --yes ppa:deadsnakes/ppa
+          apt update
+          apt install -y python3.7 python3.7-dev python3.7-venv
+
```

Comment:
py37 is already there: https://github.com/theopenlab/openlab-zuul-jobs/blob/master/playbooks/cloud-provider-openstack-test/pre.yaml 
you can either remove it or add `py37-dev` there.